src/config/dark-sites.config

```diff
@@ -238,6 +239,7 @@ elybeatmaker.com
 emanuelemilella.com
 emeraldchat.com/app
 emkc.org
+emoji.supply
```

Comment:
This site is not dark-by-default. Please remove.

![image](https://user-images.githubusercontent.com/25481501/181471595-e4f03779-628d-4710-a852-c1cef26b0b5a.png)

Reply:
Ah, sorry about that, I should have checked and targeted just https://emoji.supply/kitchen/. Fixed (I hope).